trestle/core/commands/merge.py

```diff
@@ -14,119 +14,131 @@
+            required=True
         )
 
-    def _run(self, args) -> None:
+    def _run(self, args: argparse.Namespace) -> int:
         """Merge elements into the parent oscal model."""
         log.set_log_level_from_args(args)
-        if args.list_available_elements:
-            self._list_available_elements()
-
-    def _list_available_elements(self) -> None:
-        """List element paths that can be merged from the current context."""
-        current_model, current_alias = fs.get_contextual_model_type(Path.cwd())
-
-        current_filename = f'{current_alias}.json'
-
-        self._list_options_for_merge(Path.cwd(), current_alias, current_model, current_filename)
-
-    def _list_options_for_merge(
-        self,
-        cwd: Path,
-        current_alias: str,
-        current_model: Type[OscalBaseModel],
-        current_filename: str,
-        initial_path: Path = None,
-        visited_elements: Set[str] = None
-    ):
-        """List paths that can be used in the -e option for the merge operation."""
-        if initial_path is None:
-            initial_path = cwd
-        if visited_elements is None:
-            visited_elements = set()
-
-        path_sep = '.' if current_alias else ''
-
-        # List options for merge
-        if not utils.is_collection_field_type(current_model):
-
-            malias = current_alias.split('.')[-1]
-            if cwd.is_dir() and malias != fs.extract_alias(cwd):
-                split_subdir = cwd / malias
-            else:
-                split_subdir = cwd.parent / cwd.with_suffix('').name
-
-            # Go through each file or subdirectory in the cwd
-            fields_by_alias = current_model.alias_to_field_map()
-            for filepath in Path.iterdir(split_subdir):
-                if filepath.is_file() and cwd == initial_path:
-                    continue
-
-                alias = filepath.with_suffix('').name
-                if alias in fields_by_alias:
-                    visited_element = f'{current_alias}{path_sep}{alias}'
-                    if visited_element not in visited_elements:
-                        visited_elements.add(visited_element)
-                        self.out(f"{visited_element} (merges \'{filepath.name}\' into \'{cwd / current_filename}\')")
-
-                    # If it is subdirectory, call this function recursively
-                    if Path.is_dir(filepath):
-                        self._list_options_for_merge(
-                            filepath,
-                            f'{current_alias}{path_sep}{alias}',
-                            fields_by_alias[alias].outer_type_,
-                            f'{alias}.json',
-                            initial_path=initial_path,
-                            visited_elements=visited_elements
-                        )
+        try:
+            # FIXME: Handle multiple element paths: element_paths = args.element.split(',')
```

Comment:
can we please put a check in here rather than choosing to error out badly?

Reply:
✅